### PR TITLE
:lipstick: remove: comments, dummy data

### PR DESCRIPTION
**Pull request by dfvasquez**
# Description
Comentarios o código comentario innecesario, console logs, datos dummy y TODO's terminados eliminados del código.
## Type of change
Please delete options that are not relevant.
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
# How Has This Been Tested?
Please describe the tests that you ran to verify your changes (Like web browser's or mobile devices). Provide instructions so we can reproduce.
- [x] Chrome Desktop
# Screenshots (Only if need)
#### Before this PR
image

#### After this PR
image

### CODE REVIEW

**Review by cnscheihing — APPROVED** [review]
Todo funcionando! Probado en Chrome

File: src/serviceWorker.js [hunk 1]
```diff
@@ -1,14 +1,3 @@
-// This optional code is used to register a service worker.
```
Comment (rakarachon) [inline src/serviceWorker.js:1]: Será necesario borrar este comentario? Yo al menos no cacho bien que es ServiceWorker, asique a lo mejor más adelante este comentario es util? 😅

**Review by rakarachon — COMMENTED** [review]
deje un comentario en serviceWorker.js :)

File: src/serviceWorker.js (hunk 1 above)
Comment (dfvasquez) [reply src/serviceWorker.js:1]: Oki!

**Pull request merged**